### PR TITLE
Release 3.69.0 Changelog (GA)

### PR DESCRIPTION
**Pull request by rileykarson**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 ## 3.69.0 (Unreleased)
 
+IMPROVEMENTS:
+* apigateway: allowed field `apiconfig` to change on resource `google_apigateway_gateway` ([#9139](https://github.com/hashicorp/terraform-provider-google/pull/9139))
```
Comment (c2thorn) [inline CHANGELOG.md:4]: think this is beta only

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+* compute: added "description" field to "google_compute_resource_policy" resource ([#9176](https://github.com/hashicorp/terraform-provider-google/pull/9176))
+* compute: added "instance_schedule_policy" field to "google_compute_resource_policy" resource ([#9176](https://github.com/hashicorp/terraform-provider-google/pull/9176))
+* compute: promoted field `autoscaling_policy.scaling_schedules` on `google_compute_autoscaler` and `google_compute_region_autoscaler` to ga ([#9165](https://github.com/hashicorp/terraform-provider-google/pull/9165))
+* computed: promoted `autoscaling_policy.cpu_utilization.predictive_method` on `google_compute_autoscaler` and `google_compute_region_autoscaler` to ga. ([#9156](https://github.com/hashicorp/terraform-provider-google/pull/9156))
+* dataflow: enabled updates for `google_dataflow_flex_template_job` ([#9123](https://github.com/hashicorp/terraform-provider-google/pull/9123))
```
Comment (c2thorn) [inline CHANGELOG.md:9]: `google_dataflow_flex_template_job` is currently beta only

File: CHANGELOG.md (hunk 1 above)
Comment (rileykarson) [inline CHANGELOG.md:4]:
```suggestion
```

File: CHANGELOG.md (hunk 2 above)
Comment (rileykarson) [inline CHANGELOG.md:9]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,16 @@
 ## 3.69.0 (Unreleased)
 
+IMPROVEMENTS:
+* compute: added "description" field to "google_compute_resource_policy" resource ([#9176](https://github.com/hashicorp/terraform-provider-google/pull/9176))
+* compute: added "instance_schedule_policy" field to "google_compute_resource_policy" resource ([#9176](https://github.com/hashicorp/terraform-provider-google/pull/9176))
+* compute: promoted field `autoscaling_policy.scaling_schedules` on `google_compute_autoscaler` and `google_compute_region_autoscaler` to ga ([#9165](https://github.com/hashicorp/terraform-provider-google/pull/9165))
+* computed: promoted `autoscaling_policy.cpu_utilization.predictive_method` on `google_compute_autoscaler` and `google_compute_region_autoscaler` to ga. ([#9156](https://github.com/hashicorp/terraform-provider-google/pull/9156))
```
Comment (megan07) [inline CHANGELOG.md:7]:
```suggestion
* compute: promoted `autoscaling_policy.cpu_utilization.predictive_method` on `google_compute_autoscaler` and `google_compute_region_autoscaler` to ga. ([#9156](https://github.com/hashicorp/terraform-provider-google/pull/9156))
```

**Review by megan07 — APPROVED** [review]
sorry, the `computed` was my typo. otherwise lgtm!

**Pull request merged**